queries.sql

```diff
@@ -9,3 +9,37 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE  weight_kg BETWEEN 10.4 and 17.3;
+
+SELECT * FROM animals;
+BEGIN TRANSACTION;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
> Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

- [ ] According to the requirement, it says you should do this inside a transaction, your implementation is okay, but I'll suggest you begin the transaction before selecting all from the animal table. you just need to modify your current implementation a bit to achieve this. :+1: :+1: 

```sh
BEGIN TRANSACTION;
SELECT * FROM animals;
UPDATE animals SET species='unspecified';
SELECT * FROM animals;
ROLLBACK;
```

I'll also suggest you add spaces after each query for better readability of your code. :+1: :small_airplane: